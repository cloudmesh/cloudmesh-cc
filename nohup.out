sh: 1: cd: can't cd to /mnt/c/Users/abeck/~/experiment/run-wsl
sh: 1: cd: can't cd to /mnt/c/Users/abeck/~/experiment/run-wsl
sh: 1: cd: can't cd to /mnt/c/Users/abeck/~/experiment/run-wsl
sh: 1: cd: can't cd to /mnt/c/Users/abeck/~/experiment/run-wsl
sh: 1: cd: can't cd to /mnt/c/Users/abeck/~/experiment/run-wsl
