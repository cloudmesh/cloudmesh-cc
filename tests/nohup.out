/bin/sh: 0: Illegal option -
# cloudmesh status=running progress=0 pid=26
Fri Jul  1 14:36:01 EDT 2022
# cloudmesh status=running progress=0 pid=26
-p
2-8-22
3D Objects
AppData
Application Data
Contacts
Cookies
Desktop
Documents
Downloads
ENV3
Favorites
IntelGraphicsProfiles
Links
Local Settings
Music
My Documents
NAO
NAO Robots
NTUSER.DAT
NTUSER.DAT{1c2b59c6-c5f5-11eb-bacb-000d3a96488e}.TM.blf
NTUSER.DAT{1c2b59c6-c5f5-11eb-bacb-000d3a96488e}.TMContainer00000000000000000001.regtrans-ms
NTUSER.DAT{1c2b59c6-c5f5-11eb-bacb-000d3a96488e}.TMContainer00000000000000000002.regtrans-ms
NetHood
OneDrive
Pictures
PrintHood
Recent
Saved Games
Searches
SendTo
Start Menu
Templates
Tracing
Videos
VirtualBox VMs
cm
experiment
ntuser.dat.LOG1
ntuser.dat.LOG2
ntuser.ini
run-wsl.log
run-wsl.sh
run.sh
source
# cloudmesh status=running progress=10 pid=26
# cloudmesh status=running progress=20 pid=26
LAPTOP-DK9141G4
# cloudmesh status=done progress=100 pid=26
/bin/sh: 0: Illegal option - 
+ LOG_FILE=run-wsl.log
+ tee -- run-wsl.log
+ echo # cloudmesh status=running progress=0 pid=26
# cloudmesh status=running progress=0 pid=26
+ date
Fri Jul  1 14:39:35 EDT 2022
+ echo # cloudmesh status=running progress=0 pid=26
# cloudmesh status=running progress=0 pid=26
+ ls
-p
2-8-22
3D Objects
AppData
Application Data
Contacts
Cookies
Desktop
Documents
Downloads
ENV3
Favorites
IntelGraphicsProfiles
Links
Local Settings
Music
My Documents
NAO
NAO Robots
NTUSER.DAT
NTUSER.DAT{1c2b59c6-c5f5-11eb-bacb-000d3a96488e}.TM.blf
NTUSER.DAT{1c2b59c6-c5f5-11eb-bacb-000d3a96488e}.TMContainer00000000000000000001.regtrans-ms
NTUSER.DAT{1c2b59c6-c5f5-11eb-bacb-000d3a96488e}.TMContainer00000000000000000002.regtrans-ms
NetHood
OneDrive
Pictures
PrintHood
Recent
Saved Games
Searches
SendTo
Start Menu
Templates
Tracing
Videos
VirtualBox VMs
cm
experiment
ntuser.dat.LOG1
ntuser.dat.LOG2
ntuser.ini
run-wsl.log
run-wsl.sh
run.sh
source
+ echo # cloudmesh status=running progress=10 pid=26
# cloudmesh status=running progress=10 pid=26
+ sleep 1
+ echo # cloudmesh status=running progress=20 pid=26
# cloudmesh status=running progress=20 pid=26
+ hostname
LAPTOP-DK9141G4
+ echo # cloudmesh status=done progress=100 pid=26
# cloudmesh status=done progress=100 pid=26
